ansible/configs/kni-osp/files/cloud_providers/azure_cloud_template.j2

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
Is an azure cloud provider template actually needed for this config?

Reply:
no, removing those

Reply:
@stencell can you resolve if this is ok now?